packages/fast-layouts-react/src/pane/pane.tsx

```diff
@@ -262,6 +264,9 @@ export class Pane extends Foundation<PaneHandledProps, PaneUnhandledProps, PaneS
             default:
                 break;
         }
+
+        // Fire the resize callback
+        this.onResize(e);
```

Comment:
do we also want to pass the width to this callback?

Reply:
Yeah, I think that’s valuable. Good call @janechu

Reply:
Added width as an optional param.